gio/src/lib.rs

```diff
@@ -82,6 +82,8 @@ mod simple_proxy_resolver;
 mod socket;
 pub use socket::{InputMessage, InputVector, OutputMessage, OutputVector, SocketControlMessages};
 mod socket_control_message;
+mod socket_listener;
+pub use socket_listener::SocketListenerExtManual;
```

Comment:
prelude only for traits in general

Reply:
I see, yes, the other `ExtManual` traits are in prelude, which makes sense.